src/Symfony/Bridge/Monolog/Handler/BrowserConsoleHandler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Handler;
+
+use Monolog\Handler\BrowserConsoleHandler as BaseBrowserConsoleHandler;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class BrowserConsoleHandler extends BaseBrowserConsoleHandler implements EventSubscriberInterface
+{
+    /** @var string */
+    private static $responseFormat;
```

Comment:
why is this a static property ?

Reply:
because everything else in this class is static.